0036-spsp-pull-payments/0036-spsp-pull-payments.md

```diff
@@ -0,0 +1,190 @@
+| `balance.current` | Integer String | Amount, denoted in `asset_info.code`, which remains pullable within the current frequency interval (see `frequency_info`). Once `balance.current` equals `0`, the, the client can not pull any more until `balance.current` is refilled on `timeline_info.refill_time`. |
+| `asset_info` | Object | _(OPTIONAL)_ Details about the agreement's asset. For pull payment agreements, this is the currency the pull can be made in. |
+| `asset_info.code` | String | Asset code to identify the agreement's currency. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. |
+| `asset_info.scale` | Integer | The scale of the amounts `balance.maximum` and `balance.current` are stated in (e.g. an amount of `"1000"` with a scale of `2` translates to `10.00` units of the server's asset/currency) |
+| `frequency_info` | Object | Parameters defining the recurrence of pull payments. |
+| `frequency_info.type`	| String | Frequency at which `timeline_info.refill_time` is incremented. Possible values are `DAY`, `WEEK`, `MONTH`, `YEAR`. |
+| `frequency_info.interval` | Integer | Interval associated with `frequency_info.type`. For example, if `frequency_info.type` is `WEEK` and `frequency_info.interval` is `2`, `balance.current` is refilled every 2 weeks. |
+| `timeline_info`	|	Object | Times defining the recurrence of pull payments. |
+| `timeline_info.refill_time` |	String | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, e.g. `"2019-02-10T01:01:13Z"`, representing the next time at which `balance.current` will be filled up to the amount of `balance.maximum`. |
+| `timeline_info.expiry_time` |	String | _(OPTIONAL)_ [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, e.g. `"2019-07-10T01:01:12Z"`, representing the time after which `balance.current` will not be filled up anymore, i.e., when the pull payment agreement expires. |
+| `receiver_info` | Object | _(OPTIONAL)_ Arbitrary additional information about the receiver. This field has no schema and the receiver may include any fields they choose. The field names listed below are recommended merely for interoperability purposes. |
+| `receiver_info.name` | String | _(OPTIONAL)_ Full name of the individual, company or organization the receiver represents |
+| `receiver_info.image_url` | HTTPS URL | _(OPTIONAL)_ URL where the sender can get a picture representation of the receiver |
+
+**Note:** Currency amounts are denominated as integer strings instead of native JSON numbers to avoid losing precision during JSON parsing. Applications MUST represent these numbers in a data type that has precision equal or greater than an unsigned 64-bit integer.
+
+##### Errors
+
+###### receiver Does Not Exist
+
+``` http
+HTTP/1.1 404 Not Found
+Content-Type: application/spsp4+json
+
+{
+  "id": "InvalidReceiverError",
+  "message": "Invalid receiver ID"
+}
+```
+
+## Model of Operation
+
+### Creating a token
```

Comment:
Creating a token doesn't seem to be an interoperability requirement.

The end result is a Payment Pointer that contains a bearer token. How that is created doesn't need to be standardised.

I would recommend noting that this part of the specification is not normative.

Reply:
Fair enough.